ccmlib/cmds/cluster_cmds.py

```diff
@@ -185,7 +185,7 @@ def run(self):
                         profile_options = {}
                         if self.options.profile_options:
                             profile_options['options'] = self.options.profile_options
-                    if cluster.start(verbose=self.options.debug_log, wait_for_binary_proto=self.options.binary_protocol, jvm_args=self.options.jvm_args, profile_options=profile_options) is None:
+                    if cluster.start(verbose=self.options.debug_log, wait_for_binary_proto=self.options.binary_protocol, jvm_args=self.options.jvm_args, profile_options=profile_options, allow_root=self.options.allow_root) is None:
```

Comment:
This breaks `ccm create` because this command has a different options parser than ClusterStart:

```
  File "build/bdist.linux-x86_64/egg/ccmlib/cmds/cluster_cmds.py", line 188, in run
AttributeError: Values instance has no attribute 'allow_root'
```